manifests/init.pp

```diff
@@ -119,9 +120,9 @@
 #   include hdm
 class hdm (
   # installation parameter
+  String[1]                     $version               = '1.0.1',
```

Comment:
Each change here will require a new major version. Unless this "hdm" is something that is unlikely to evolve, it is probably better to **not** set a default value and put that responsibility on the end user.

Reply:
I'm not sure about this. If a user want's to run the latest release, they need to update their local code/hiera. It doesn't matter if the module has no version vs an old version. But with a default version in the module the user can get started with `include hdm` and we've that pattern in most other modules, for example puppet/prometheus. We don't update the modules to always ship the latest release.